src/Database/Attach/File.php

```diff
@@ -86,8 +86,11 @@ public function fromPost($uploadedFile)
         $this->file_size = $uploadedFile->getClientSize();
         $this->content_type = $uploadedFile->getMimeType();
         $this->disk_name = $this->getDiskName();
-
-        $this->putFile($uploadedFile->getRealPath(), $this->disk_name);
+        $realpath = ( empty(trim($uploadedFile->getRealPath())) 
```

Comment:
For some reason getRealPath is empty on IIS. A test could be implemented  to test if it's empty(just as i'm doing here) It simply won't save properly, throwing whatever Exception it does further in the pipeline when it can't find the actual file because the filepath is empty to the C:\windows\tmp directory or something.